FirstClasses/fibonacci_sum/src/fibonacci_sum.erl

```diff
@@ -7,4 +7,11 @@ fibonacci_sum() ->
     fibonacci_sum(1, 1, 0, Limit).
 
 fibonacci_sum(_Prev, _Curr, _Sum, _Limit) ->
-    1.
+    case _Sum of
```

Comment:
The very tiny implementation 👍 I think you could have removed `_Sum` parameter completely. Which would be something like this:
```erlang
fibonacci_sum(_Prev, _Curr, _Limit) ->
    case 1 of
        _ when _Curr =< _Limit andalso _Curr rem 2 == 0 ->
            _Curr + fibonacci_sum(_Curr, _Prev + _Curr, _Limit);
        _ when _Curr =< _Limit ->
            fibonacci_sum(_Curr, _Prev + _Curr, _Limit);
        _ ->
            0
        end.
```
Also using function head matching will simplify the code even more:
```erlang
fibonacci_sum(_Prev, _Curr, _Limit) when _Curr =< _Limit andalso _Curr rem 2 == 0 ->
            _Curr + fibonacci_sum(_Curr, _Prev + _Curr, _Limit);
fibonacci_sum(_Prev, _Curr, _Limit) when _Curr =< _Limit ->
            fibonacci_sum(_Curr, _Prev + _Curr, _Limit);
fibonacci_sum(_Prev, _Curr, _Limit) ->
            0.
```
Your solution works great, the motivation behind storing a `Sum` was to use tail recursion, like this:
```erlang
fibonacci_sum(Prev, Curr, Sum, Limit) when Curr =< Limit andalso Curr rem 2 == 0 ->
    fibonacci_sum(Curr, Prev + Curr, Sum + Curr, Limit);
fibonacci_sum(Prev, Curr, Sum, Limit) when Curr =< Limit ->
    fibonacci_sum(Curr, Prev + Curr, Sum, Limit);
fibonacci_sum(_, _, Sum, _) -> Sum.
```